### PR TITLE
Fixed issue #4998 and refactored, removed deprecated code

### PR DESCRIPTION
**Pull request by Pjoottr**
If you entered the site being a second user the URL structure would switch, from `stadia.com/home` to `stadia.com/u/1/home`, fixed by checking if the URL included home, then expect the user was at the home page, rather than looking for an exact URL match.

Before:
image

After:
image

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Fixes #4998

File: websites/S/Stadia/presence.ts [hunk 1]
```diff
@@ -10,10 +10,10 @@ let gameName: HTMLElement,
 presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
     largeImageKey: "logo",
-    startTimestamp: browsingStamp
+    startTimestamp: browsingStamp,
```
Comment (EncryptedDev) [inline websites/S/Stadia/presence.ts:13]: Revert this change

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
bump version

File: websites/S/Stadia/presence.ts (hunk 1 above)
Comment (Pjoottr) [reply websites/S/Stadia/presence.ts]: All done 👍

**Review by EncryptedDev — COMMENTED** [review]
You still need to bump the version in `metadata.json`

**Comment by Pjoottr** [comment]
Oh yes, I'm sorry forgot about that. It would be 1.0.10 right?

File: websites/S/Stadia/dist/metadata.json [hunk 2]
```diff
@@ -4,6 +4,12 @@
     "name": "Alexx",
     "id": "604453403640463371"
   },
+  "contributors": [
+    {
+      "name": "Pjoottr",
+      "id": "906309000465903697"
+    }
+  ],
```
Comment (Slowlife01) [inline websites/S/Stadia/dist/metadata.json:12]: image
Not exactly a quarter yet if you add support for library path then sure

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**